src/logging.js

```diff
@@ -45,9 +45,17 @@ function handle (level, args) {
   if (!log) {
     console.error("Log not initialised"); // eslint-disable-line no-console
   }
-  // Take the first arg as context
-  // apply util.format to the second arg, with the remaining args as input
-  const message = args[0] + " " + util.format.apply(null, args.slice(1));
+
+  // If the first arg does NOT contain '%', assume it is context and append it onto the
```

Comment:
Should we be checking for args.length >= 2 first? If you've got a simple log line, that could be picked up as a context with no message.